### PR TITLE
Add files needed for ADO security tools pipeline

### PR DESCRIPTION
**Pull request by VeryEarly**
## Description
files needed for security tools CI pipeline

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: tools/SecurityTools/RunBinSkim.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$repoRoot = "$PSScriptRoot/../.."
+$binSkim = (Get-Item -Path "$repoRoot/tools/SecurityTools/Microsoft.CodeAnalysis.BinSkim*/tools/*/win-x64/BinSkim.exe" | Select-Object -First 1).FullName
+$dllBaseFolder = "$repoRoot/artifacts/Debug"
```
Comment (erich-wang) [inline tools/SecurityTools/RunBinSkim.ps1]: Do we only run security for debug configuration?

File: tools/SecurityTools/CredScanSuppressions.json [hunk 2]
```diff
@@ -0,0 +1,1097 @@
+{
```
Comment (erich-wang) [inline tools/SecurityTools/CredScanSuppressions.json]: Do you just copy the files from master branch? If yes, have you made any change after copying? I would suggest use first commit for exactly same files from master branch, then submit your changes using following commits, to reduce review time.

File: tools/SecurityTools/CredScanSuppressions.json [hunk 3]
```diff
@@ -0,0 +1,1097 @@
...
+  "tool": "Credential Scanner",
+  "suppressions": [
+    {
+      "file": "\\documentation\\development-docs\\azure-powershell-developer-guide.md",
```
Comment (erich-wang) [inline tools/SecurityTools/CredScanSuppressions.json]: should we use / instead of \\?

File: tools/SecurityTools/CredScanSuppressions.json (hunk 2 above)
Comment (VeryEarly) [reply tools/SecurityTools/CredScanSuppressions.json]: first commit have the original file copied from master branch,
changes was made in commit:
65dadc63c676c125caa108483af300137478e045

Comment (VeryEarly) [reply tools/SecurityTools/CredScanSuppressions.json]: > Do you just copy the files from master branch? If yes, have you made any change after copying? I would suggest use first commit for exactly same files from master branch, then submit your changes using following commits, to reduce review time.

first commit have the original file copied from master branch,
changes was made in commit:
65dadc6

File: tools/SecurityTools/CredScanSuppressions.json (hunk 3 above)
Comment (VeryEarly) [reply tools/SecurityTools/CredScanSuppressions.json]: this pattern is in the original file from master branch,
new added fields followed this pattern exactly.

File: tools/SecurityTools/RunBinSkim.ps1 (hunk 1 above)
Comment (VeryEarly) [reply tools/SecurityTools/RunBinSkim.ps1]: in "azure-powershell-security-tools": 
[https://dev.azure.com/azure-sdk/internal/_build?definitionId=248&_a=summary](url)

It runs for debug configuration only; build arguments:
"build.proj /t:Build;GenerateHelp"

**Pull request merged**